test/controllers/contactPreference/ChangeEmailControllerSpec.scala

```diff
@@ -0,0 +1,336 @@
+                common.SessionKeys.validationEmailKey -> testValidationEmail,
+                common.SessionKeys.prepopulationEmailKey -> testValidEmail)
+              )
+            }
+            lazy val document = Jsoup.parse(bodyOf(result))
+
+            "return 200" in {
+              status(result) shouldBe Status.OK
+            }
+
+            "return HTML" in {
+              contentType(result) shouldBe Some("text/html")
+              charset(result) shouldBe Some("utf-8")
+            }
+
+            "prepopulate the form with the previously entered form value" in {
+              document.select("input").attr("value") shouldBe testValidEmail
+            }
+
+            "not call the VatSubscription service" in {
+              verify(mockVatSubscriptionService, never())
+                .getCustomerInfo(any[String])(any[HeaderCarrier], any[ExecutionContext])
+            }
+          }
+        }
+
+        "there is no email in session" when {
+
+          "the customerInfo call succeeds" should {
+
+            lazy val result = target().show(request)
+            lazy val document = Jsoup.parse(bodyOf(result))
+
+            "return 200" in {
+              status(result) shouldBe Status.OK
+            }
+
+            "return HTML" in {
+              contentType(result) shouldBe Some("text/html")
+              charset(result) shouldBe Some("utf-8")
+            }
+
+            "prepopulate the form with the customerInfo result" in {
+              document.select("input").attr("value") shouldBe testValidEmail
+            }
+          }
+
+          "the customerInfo call fails" should {
+
+            lazy val result = target(Left(ErrorModel(
+              Status.NOT_FOUND,
+              "error"
+            ))).show(request)
+
+            "return 404" in {
+              status(result) shouldBe Status.INTERNAL_SERVER_ERROR
+            }
+
+            "return HTML" in {
+              contentType(result) shouldBe Some("text/html")
+              charset(result) shouldBe Some("utf-8")
+            }
+          }
+        }
+      }
+
+
+
+      "a user is does not have a valid enrolment" should {
+
+        lazy val result = target().show(request)
+
+        "return 403" in {
+          mockIndividualWithoutEnrolment()
+          status(result) shouldBe Status.FORBIDDEN
+        }
+
+        "return HTML" in {
+          contentType(result) shouldBe Some("text/html")
+          charset(result) shouldBe Some("utf-8")
+        }
+      }
+
+
+      "a user is not logged in" should {
+
+        lazy val result = target().submit(request)
+
+        "return 401" in {
+          mockMissingBearerToken()
+          status(result) shouldBe Status.UNAUTHORIZED
+        }
+
+        "return HTML" in {
+          contentType(result) shouldBe Some("text/html")
+          charset(result) shouldBe Some("utf-8")
+        }
+      }
+
+      "the inflight predicate is not mocked out and there is nothing in session" should {
```

Comment:
I'm not clear on what this test is doing differently than the others? The `inflightTarget` val below looks to be creating a controller in the same way that the `target` function does in all the other tests.